package.json

```diff
@@ -35,7 +35,7 @@
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.0",
     "ember-cli-jsoneditor": "0.1.4",
-    "ember-cli-moment-shim": "3.3.3",
+    "ember-cli-moment-shim": "3.3.0",
```

Comment:
We had to downgrade?

Reply:
3.3.0 is the version that ember-cli-bootstrap-datetimepicker uses. I can try it with 3.3.3?